src/main/java/net/sourceforge/pmd/buildtools/surefire/junit/TestExecutionListener.java

```diff
@@ -30,7 +34,7 @@ class TestExecutionListener implements org.junit.platform.launcher.TestExecution
 
     private final TestReportListener<TestOutputReportEntry> testReportListener;
 
-    public TestExecutionListener(TestReportListener<TestOutputReportEntry> testReportListener) {
```

Comment:
- https://github.com/palantir/palantir-java-format/pull/1342

Reply:
:question: Please comment in full sentences, thanks.